tools/proofers/for_mentors.php

```diff
@@ -110,56 +110,113 @@
 
 // ---------------------------------------------------------------
 
-// For each mentorable project (in this round),
-// show a summary (one line per mentee)
-// and then a listing (one line per page).
+$mentored_round = $mentoring_round->mentee_round;
 
+// output a table of contents with links to anchors on this page
 echo "<p>";
-echo sprintf(_("Pages available to Mentors in round %s."), "<b>" . $mentoring_round->id . "</b>");
+echo sprintf(_("Projects with pages available to Mentors in round %s."), "<b>$mentoring_round->id</b>");
 echo " ";
 echo _("Oldest project listed first.");
 echo "</p>";
 
+$projects_available = get_beginner_projects_in_state($mentoring_round->project_available_state);
+if($projects_available)
+{
+    echo "<ol>";
+    foreach($projects_available as $proj_obj)
+    {
+        echo "<li><a href='#$proj_obj->projectid'>";
+        echo output_project_label($proj_obj->nameofwork, $proj_obj->authorsname);
+        echo "</a></li>";
+    }
+    echo "</ol>";
+}
+else
+{
+    echo "<p><i>" . _("none") . "</i></p>";
+}
 
-$mentored_round = $mentoring_round->mentee_round;
-$result = mysqli_query(DPDatabase::get_connection(), project_sql($mentoring_round));
-while ($proj =  mysqli_fetch_object($result))
+// output a listing of projects in this mentoring round that are in a waiting state
+echo "<p>";
+echo sprintf(_("Projects for Mentors waiting in round %s."), "<b>$mentoring_round->id</b>");
```

Comment:
Suggested rewording:
"Projects for Mentors, waiting to be released into round %s."

Reply:
Updated!